project/settings.py

```diff
@@ -291,8 +291,7 @@
     STATICFILES_LOCATION = "/static"
 
     MEDIAFILES_LOCATION = "/media"
-
-    EXPERIMENT_LOCATION = "/experiments"
+    EXPERIMENT_LOCATION = "experiments"
```

Comment:
This could be moved out of the if/else statement since it's now the same value for cloud and local storage. Not a big deal though.